src/data/menu.json

```diff
@@ -0,0 +1,37 @@
+{
+	"food" : [
+	
+		{
+      "disch" : "Hamburgare med pommes frites",
+		  "price" : "120 kr",
```

Comment:
Här verkar det vara en felstavning! Kanske en korrigering från "disch" till "dish"? Annars ser det bra ut :)

Reply:
Tack för påminnelsen. Jag har ändrat det redan i VScode. Ska bara git pull i main och sedan pusha in ny JSON-fil.

Reply:
Har nu fixat problemet med felstavningen och pushat nytt.